src/Symfony/Component/Stopwatch/Section.php

```diff
@@ -110,23 +110,23 @@ public function setId(string $id)
      *
      * @return StopwatchEvent The event
      */
-    public function startEvent(string $name, ?string $category)
+    public function startEvent(string $internalId, ?string $label)
```

Comment:
I would use `$id`

Reply:
I didn't want to do as there is already ID used in https://github.com/symfony/symfony/blob/b2244511bfb6f9b09693fb80e779db224c41a56e/src/Symfony/Component/Stopwatch/Section.php#L61-L83